### ISSUE TITLE
Is it possible to show the document structure by default when opening the pdf?

### ISSUE THREAD

**Issue by dvag-vibula-kanakalingam** [stephanrauh/ngx-extended-pdf-viewer#1487]
![image](https://user-images.githubusercontent.com/85543601/184152700-35a3b5a8-19a0-4489-a7e2-b09f8b77d785.png)

**Comment by stephanrauh** [comment]
Yeah, sure. You can't do it now, because it's one of the simple ideas that are obvious but nonetheless never crossed my mind. But I can implement the feature for you in version 15.0.0.

**Comment by stephanrauh** [comment]
Your feature has landed with version 15.2.0.

Enjoy!
Stephan

**Comment by stephanrauh** [comment]
There's a small inconsistency of the implementation - but I guess it's good it's there. When you load a new file by changing the `[src]` attribute, the sidebar view is always reset to show the thumbnails. I don't want to remove this behavior because every PDF has thumbnails, but few PDF files have any of the other three sidebar view. So it's better to stick to the default.

However, if you know for sure that your PDF has - say - layers, you can set the attribute with a short timeout:

```typescript
public load(file: string): void {
  const view = this.activeSidebarView;
  this.src=file;
  // loading a file resets the default view
  setTimeout(() => this.activeSidebarView = view, 150);
}
```
